e2e/tests/transfer/base_test.go

```diff
@@ -196,62 +92,24 @@ func (s *TransferTestSuite) TestMsgTransfer_Fails_InvalidAddress() {
 		expected := testvalues.StartingTokenAmount
 		s.Require().Equal(expected, actualBalance)
 	})
-}
-
-func (s *TransferTestSuite) TestMsgTransfer_Timeout_Nonincentivized() {
-	t := s.T()
-	ctx := context.TODO()
-
-	relayer, channelA := s.SetupChainsRelayerAndChannel(ctx, s.TransferChannelOptions())
-	chainA, _ := s.GetChains()
-	chainAWallet := s.CreateUserOnChainA(ctx, testvalues.StartingTokenAmount)
-	chainBWallet := s.CreateUserOnChainB(ctx, testvalues.StartingTokenAmount)
-
-	chainBWalletAmount := ibc.WalletAmount{
-		Address: chainBWallet.FormattedAddress(), // destination address
-		Denom:   chainA.Config().Denom,
-		Amount:  sdkmath.NewInt(testvalues.IBCTransferAmount),
-	}
-
-	t.Run("IBC transfer packet timesout", func(t *testing.T) {
-		tx, err := chainA.SendIBCTransfer(ctx, channelA.ChannelID, chainAWallet.KeyName(), chainBWalletAmount, ibc.TransferOptions{Timeout: testvalues.ImmediatelyTimeout()})
-		s.Require().NoError(err)
-		s.Require().NoError(tx.Validate(), "source ibc transfer tx is invalid")
-		time.Sleep(time.Nanosecond * 1) // want it to timeout immediately
-	})
-
-	t.Run("tokens are escrowed", func(t *testing.T) {
-		actualBalance, err := s.GetChainANativeBalance(ctx, chainAWallet)
-		s.Require().NoError(err)
-
-		expected := testvalues.StartingTokenAmount - testvalues.IBCTransferAmount
-		s.Require().Equal(expected, actualBalance)
-	})
-
-	t.Run("start relayer", func(t *testing.T) {
-		s.StartRelayer(relayer)
-	})
-
-	t.Run("ensure escrowed tokens have been refunded to sender due to timeout", func(t *testing.T) {
-		// ensure destination address did not receive any tokens
-		bal, err := s.GetChainBNativeBalance(ctx, chainBWallet)
-		s.Require().NoError(err)
-		s.Require().Equal(testvalues.StartingTokenAmount, bal)
 
-		// ensure that the sender address has been successfully refunded the full amount
-		bal, err = s.GetChainANativeBalance(ctx, chainAWallet)
-		s.Require().NoError(err)
-		s.Require().Equal(testvalues.StartingTokenAmount, bal)
+	t.Run("stop relayer", func(t *testing.T) {
+		s.StopRelayer(ctx, relayer)
 	})
 }
 
 // TestSendEnabledParam tests changing ics20 SendEnabled parameter
 func (s *TransferTestSuite) TestSendEnabledParam() {
 	t := s.T()
+
```

Comment:
are we missing a `t.Parallel()`?

Reply:
I tried it but seems it should only be used in certain situations. Like the problem you predicted when you created the issue, the tests interfere with each other so they don't work properly.

Reply:
Okay! In this case, maybe this params test can go in its own file, and it's own test suite, so we are not waiting on this test to run before starting others.